src/auth/user.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Req, UseGuards } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { ApiBearerAuth, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+@ApiTags('User')
+@Controller('user')
```

Comment:
rest api에서는 path명을 복수형으로 표현합니다.

```suggestion
@Controller('users')
```

Reply:
왜 복수형을 써야 하는지 궁금해서 관련 내용을 좀 찾아봤어요 리소스를 컬렉션 단위로 다루는 REST 개념상 복수형이 더 적절하고 /users/:id/posts 같은 관계 표현도 자연스럽게 된다는 점을 새로 알아가게 되었습니다 좋은 피드백 감사합니다!